geps/gep-3792/index.md

```diff
@@ -0,0 +1,190 @@
+# GEP-3792: External Gateways
+
+* Issue: [#3792](https://github.com/kubernetes-sigs/gateway-api/issues/3792)
+* Status: Provisional
+
+(See [status definitions](../overview.md#gep-states).)
+
+## User Story
+
+**[Chihiro] and [Ian] want a way for out-of-cluster Gateways to be able to
```

Comment:
It's probably important to note that this proposal would also allow a way for implementations that do work in-cluster but don't work well with existing Service Mesh identity injection methods to more usefully interoperate there.

Yes, Cilium is an example here - if there's a way to configure a cluster-external Gateway to talk to a TLS-based Service Mesh, then that method can also be used for Cilium's Gateway API support.

Reply:
It's an _extremely_ good point, though. Let me see if I can figure out how to work that in.

Reply:
Mentioned in background.